content/en/tracing/manual_instrumentation/go.md

```diff
@@ -93,11 +211,95 @@ func handler(w http.ResponseWriter, r *http.Request) {
 }
 ```
 
+## Trace Client & Agent Configuration
+
+There are additional configurations possible for both the tracing client and Datadog Agent for context propagation with B3 Headers, as well as to exclude specific Resources from sending traces to Datadog in the event these traces are not wanted to count in metrics calculated, such as Health Checks.
+
+### B3 Headers Extraction and Injection
+
+The Datadog APM tracer supports [B3 headers extraction][6] and injection for distributed tracing.
+
+Distributed headers injection and extraction is controlled by
+configuring injection/extraction styles. Two styles are
+supported: `Datadog` and `B3`.
+
+Configure injection styles using the environment variable
+`DD_PROPAGATION_STYLE_INJECT=Datadog,B3`
+
+Configure extraction styles using the environment variable
+`DD_PROPAGATION_STYLE_EXTRACT=Datadog,B3`
+
+The values of these environment variables are comma separated lists of
+header styles that are enabled for injection or extraction. By default only
+the `Datadog` extraction style is enabled.
+
+If multiple extraction styles are enabled, extraction attempts are made
+in the order that those styles are specified. The first successfully
+extracted value is used.
+
+### Resource filtering
+
+The Agent can be configured to exclude a specific Resource from Traces sent by the Agent to the Datadog application. To prevent the submission of specific Resources, use the `ignore_resources` setting in the `datadog.yaml` file . This setting enables the creation of a list containing one or more regular expressions, which instructs the Agent to filter out Traces based on their Resource name.
```

Comment:
```suggestion
The Agent can be configured to exclude a specific resource from traces sent by the Agent to the Datadog application. To prevent the submission of specific resources, use the `ignore_resources` setting in the `datadog.yaml` file . This setting enables the creation of a list containing one or more regular expressions, which instructs the Agent to filter out traces based on their resource name.
```